orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -359,14 +360,76 @@ void addObjectMethodsForTraining(py::module& m, ExecutionProviderRegistrationFn
       .def("__len__", [](const std::vector<OrtValue>& v) { return v.size(); })
       .def("__iter__", [](const std::vector<OrtValue>& v) {
         return py::make_iterator(v.cbegin(), v.cend());
-      },
-           py::keep_alive<0, 1>())
+       }, py::keep_alive<0, 1>())
       .def("__getitem__", [](const std::vector<OrtValue>& v, const size_t idx) {
         return v.at(idx);
       })
       .def("dlpack_at", [](std::vector<OrtValue>* v, const size_t idx) {
         return py::reinterpret_steal<py::object>(ToDlpack(v->at(idx)));
-      });
+      })
+      .def("to_dlpack", [](const std::vector<OrtValue>& v, py::object to_tensor) -> py::list {
```

Comment:
I still think we should rename this one. Even we still want to keep the dlpack, it should also be "to_dlpacks".